detail.css

```diff
@@ -0,0 +1,77 @@
+.myimage
+{
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+#image_sidebar
+{
+  width:95px;
+  float:left;
+  display:flex;
+}
+
+#image_sidebar ul
+{
+  padding:0px;
+  margin:0px;
+}
+
+#image_sidebar ul li
```

Comment:
nice use of inheriting css classes (is inheritance in css even a thing?)

Reply:
Yes ma'am

Reply:
Thank you ma'am.